.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,15 @@ Fixes #
 
 **Special notes for your reviewer**:
 
+**Checklist**
+
+This checklist is not enforcing, but it's a reminder of items that could be relevant to this PR:
+
+- [ ] Design: A design document was considered and is present (link) or not required
+- [ ] Test: Unit and e2e tests were balanced each other and the right approach chosen
```

Comment:
We should add a link to the [reviewer guide "good enough section"](https://github.com/kubevirt/kubevirt/blob/421a6a3acb79204f8fab1e46cb061a33a8f6dbd7/docs/reviewer-guide.md#when-is-a-pr-good-enough) here.

Or we just copy the part into this one.

```suggestion
- [ ] Testing:
  - Unit and e2e tests struck a balance between each other, leaning towards more unit tests and the right approach was chosen
  - Hint: new code requires new unit tests.
  - Hint: new features and bug fixes require at least one e2e test (the core case must be tested).
```

BTW could you elaborate in what you mean with "the right approach was chosen"

Reply:
> BTW could you elaborate on what you mean with "the right approach was chosen"?

I just tried to appeal at the reader to really think about their decision by including the word *right*. I wanted to them to think twice about what test type to use. But your suggestion is nice, and I'll updat ethe pr - it's much more transparent about what is expected.